cmd/xl-storage.go

```diff
@@ -785,6 +785,16 @@ func (s *xlStorage) ListDir(ctx context.Context, volume, dirPath string, count i
 // or multiple objects.
 func (s *xlStorage) DeleteVersions(ctx context.Context, volume string, versions []FileInfo) []error {
 	errs := make([]error, len(versions))
+
+	select {
```

Comment:
@harshavardhana we already have this in cmd/xl-storage-disk-id-check.go

Reply:
we do @vadmeste ? let me remove it.